tests/acceptance/AbstractContainerTest.php

```diff
@@ -18,12 +18,25 @@ public function testItCanBeConfiguredFromFiles()
 
         $this->createJSONConfigFile('config.json', $config);
 
-        ConfigureContainer::fromFiles(
-            $this->container,
-            [$this->getTestPath('*')],
-            ['config_prefix' => 'settings']
-        );
+        Configurator::apply()
+            ->configFromFiles($this->getTestPath('*'))
+            ->to($this->container);
 
-        $this->assertSame('example-value', $this->container->get('settings.example-key'));
+        $this->assertSame('example-value', $this->container->get('config.example-key'));
+    }
+
+    public function testItAddToConfigUsingFiles()
+    {
+        $config = ['keyB' => 'valueB'];
+
+        $this->createJSONConfigFile('config.json', $config);
+
+        Configurator::apply()
+            ->configFromArray(['keyA' => 'valueA', 'keyB' => 'valueX'])
+            ->configFromFiles($this->getTestPath('*'))
+            ->to($this->container);
```

Comment:
@tomphp 

Not sure, but how about we require test classes to implement a `createContainer()` method, and use its return value instead?